actions/get-vault-secrets/action.yaml

```
@@ -62,6 +62,14 @@ runs:
         COMMON_SECRETS: ${{ inputs.common_secrets }}
         REPO: ${{ github.repository }}
 
+    - id: get-github-jwt-token
+      uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
+      with:
+        script: |
+          const jwt = await core.getIDToken();
```

let's set an audience though, so that on the other side we can validate the audience. it prevents leaked tokens from other workflows being given to this endpoint.

```suggestion
          const jwt = await core.getIDToken("vault-github-actions-grafana-${{ inputs.instance }}");
```

(the given audience is an example - you can change it if you want 🙂)